test/mpi/main.cc

```diff
@@ -505,14 +518,15 @@ void ProcessArgs(int argc, char** argv)
 
 int main(int argc, char *argv[])
 {
-    std::chrono::steady_clock::time_point begin =
-        std::chrono::steady_clock::now();
-    int failed = 0;
+    int failed                       = 0;
+    int total_done_skipped_failed[4] = {0};
+    std::chrono::steady_clock::time_point begin;
     int size, required, provided, completed, rank;
     UccTestMpi *test;
     MPI_Request req;
     std::string err;
 
+    begin = std::chrono::steady_clock::now();
```

Comment:
why do we not assign it when it is declared ?

Reply:
just because alignment looks bad if move it to declaration block